src/agri_ai/tools/agricultural_tools.py

```
@@ -202,16 +213,28 @@ async def _arun(self, field_name: str, crop: str, issue: Optional[str] = None) -
+            # Get recent usage history for this field
+            try:
+                recent_usage = await self.agri_db.get_recent_material_usage(field_name)
+                if recent_usage:
+                    rec_info.append("\n最近の使用履歴:")
+                    for usage in recent_usage[-2:]:  # Last 2 usages
+                        date = usage.get("使用日", "N/A")
+                        material = usage.get("資材名", "N/A")
+                        amount = usage.get("使用量", "N/A")
+                        unit = usage.get("単位", "")
+                        rec_info.append(f"  - {date}: {material} {amount}{unit}")
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)